constants.py

```diff
@@ -0,0 +1,6 @@
+# we measured and these are the pixel per degree in these resolutions
+X_PIXELS_PER_DEGREE = {640: 10.6, 320: 5.3}
+Y_PIXELS_PER_DEGREE = {480: 10.6, 240: 5.3}
+

```

Comment:
Why do we need the below constants

Reply:
i wanted them to be hard coded and not change them so i out them in another file